instructor/client_mistral.py

```diff
@@ -24,7 +24,8 @@ def from_mistral(
 
 
 def from_mistral(
-    client: mistralai.client.MistralClient | mistralaiasynccli.MistralAsyncClient,
+    # deprecated from MistralClient, MistralAsyncClient to Mistral (https://github.com/mistralai/client-python/blob/main/MIGRATION.md)
+    client: mistralai.client.Mistral | mistralaiasynccli.Mistral,
```

Comment:
yes sir

Reply:
you can checkout this readme file by mistral client https://github.com/mistralai/client-python/blob/main/MIGRATION.md

Reply:
i guess that's not enough because the chat completions also changed, now there is also complete_async and complete() instead of different clients so the line 37 assertions is indeed deprecated too

Reply:
yeah... I think this might not fix everything since mistral has now changed all their methods. Happy to take a PR for this!